index.js

```diff
@@ -0,0 +1,377 @@
+const express = require('express'); // pull in express
+const cors = require('cors');
+const server = express(); // instantiate server
+const logger = require('morgan');
+const db = require('./data/helpers/userDb');
+const postDb = require('./data/helpers/postDb');
+const port = 5201;
+
+
+const capitalize = (req, res, next) => {
+  const { name } = req.body;
+  req.body.name = req.body.name.charAt(0).toUpperCase() + req.body.name.slice(1);
+  next();
+};
+
+const textRecognize = (req, res, next) => {
+  const newText = req.body.text;
+  req.text = newText
```

Comment:
Not sure what this is meant to do? Doesn't appear to be in use anyways, so maybe just some experimenting?